src/App.vue

```diff
@@ -1,37 +1,21 @@
+<script setup lang="ts">
+import NavComponent from "./components/Navigation.vue";
+</script>
+
 <template>
-  <div>
-    <nav-component class="mb-4" />
+  <v-app>
+    <nav-component />
 
-    <b-container fluid class="main-container">
-      <b-alert
-        :show="$store.getters.hasError"
-        variant="danger"
-        dismissible
-        @dismissed="clearError"
-      >
-        {{ $store.state.error }}
-      </b-alert>
+    <v-main style="margin: 42px;">
```

Comment:
raw styles are not very managabel, better would be to use custom classes, makes it clearer too, what was intended here